--- /home/uri/uriva/abstract-bot-api/src/fbUtils.test.ts
+++ /home/uri/uriva/abstract-bot-api/src/fbUtils.test.ts
@@ -151,4 +151,9 @@
   },
+  {
+    testName: "handles urls properly",
+    input: '1. **<a href="https://d3u0tzju9qaucj.cloudfront.net/857042ed-4902-4984-b3ad-8ffee8c2268e/3fc5bb0b-11e2-4f96-8993-46252eeea094.png">שוט פתיחה - המארח עם יומן הנייר</a>**<br>המארח יושב בסלון שטוף שמש, נראה טיפה עמוס.',
+    output: '1. **שוט פתיחה - המארח עם יומן הנייר - https://d3u0tzju9qaucj.cloudfront.net/857042ed-4902-4984-b3ad-8ffee8c2268e/3fc5bb0b-11e2-4f96-8993-46252eeea094.png**\nהמארח יושב בסלון שטוף שמש, נראה טיפה עמוס.',
+  },
 ];
